src/appengine/libs/request_cache.py

```diff
@@ -40,16 +39,11 @@ def cache(self):
 
     key = '__cache:' + self._cache_key
 
-    if flask.request:
-      backing = getattr(flask.g, key, None)
-    else:
-      backing = getattr(request, key, None)
+    cache_backing = auth.get_cache_backing()
```

Comment:
get_cache_backing() shouldn't be part of the `auth` module.  It should be part of this module instead.